snapshots/output/syntax/src/accessors.ts

```diff
@@ -1,81 +1,81 @@
   class C {
 //      ^ definition syntax 1.0.0 src/`accessors.ts`/C#
-//      documentation ```ts\nC\n```
+//      documentation ```ts\nclass C\n```
     _length: number = 0
 //  ^^^^^^^ definition syntax 1.0.0 src/`accessors.ts`/C#_length.
-//  documentation ```ts\nnumber\n```
+//  documentation ```ts\n(property) _length: number\n```
     get length(): number {
 //      ^^^^^^ definition syntax 1.0.0 src/`accessors.ts`/C#`<get>length`().
-//      documentation ```ts\nnumber\n```
+//      documentation ```ts\nget length: number\n```
       return this._length
 //                ^^^^^^^ reference syntax 1.0.0 src/`accessors.ts`/C#_length.
     }
     set length(value: number) {
 //      ^^^^^^ definition syntax 1.0.0 src/`accessors.ts`/C#`<get>length`().
-//      documentation ```ts\nnumber\n```
+//      documentation ```ts\nget length: number\n```
```

Comment:
This is incorrect.

Reply:
Hmm, there seems to be something consistently wrong with how we're handling accessors. Notice the

```
definition syntax 1.0.0 src/`accessors.ts`/C#`<get>length`().
``` 

earlier, it should be `<set>` and not `<get>`. I'll fix the accessors in a separate PR.